legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/relational/RelationalLSPGrammarExtension.java

```diff
@@ -205,7 +206,9 @@ protected void collectCommands(SectionState sectionState, PackageableElement ele
     @Override
     public Iterable<? extends LegendExecutionResult> execute(SectionState section, String entityPath, String commandId, Map<String, String> executableArgs, Map<String, Object> inputParams)
     {
-        return GENERATE_MODEL_MAPPING_COMMAND_ID.equals(commandId) ? generateModelsFromDatabaseSpecification(section, entityPath) : super.execute(section, entityPath, commandId, executableArgs, Map.of());
+        return GENERATE_MODEL_MAPPING_COMMAND_ID.equals(commandId)
+               ? generateModelsFromDatabaseSpecification(section, entityPath)
+               : FunctionExecutionSupport.execute(this, section, entityPath, commandId, executableArgs, inputParams);
```

Comment:
Similar to other comments, we dont need this on relational.

Reply:
Makes sense. Reverted the change to this file.